drivers/modem/hl7800.c

```diff
@@ -1070,6 +1072,32 @@ void mdm_hl7800_generate_status_events(void)
 	hl7800_unlock();
 }
 
+/** @ref /zephyr/samples/subsys/logging/logger/src/main.c */
+static int log_source_id_get(const char *name)
```

Comment:
Perhaps it'd be better to expose this functionality from the logger module itself instead of copying the function around? Similar is used not only in the mentioned sample, but in `log_cmds.c` as well. @nordic-krch Any opinion on this?

Reply:
I agree with @rlubos here. @nordic-krch could you also weigh in please?

Reply:
@nordic-krch any thoughts? I'd like to get some movement on this. Thanks!